.github/workflows/continuous-integration-e2e.yaml

```diff
@@ -83,10 +83,10 @@ jobs:
         yarn workspace @cardano-sdk/e2e test:projection
         yarn workspace @cardano-sdk/e2e test:pg-boss
       env:
-        STAKE_POOL_PROVIDER_URL: 'http://localhost:4000/stake-pool'
+        STAKE_POOL_PROVIDER_URL: 'http://localhost:4000/'
 
     - name: 🔬 Test - e2e - wallet - typeorm stake pool provider
       run: |
-        yarn workspace @cardano-sdk/e2e test:providers
+        yarn workspace @cardano-sdk/e2e test:providers -t HandleProvider
```

Comment:
```suggestion
        yarn workspace @cardano-sdk/e2e test:providers
```

Reply:
Currently `HandleProvider` is the only test under `test:providers`.

Since here we want to test the _typeorm stake pool provider_ we neet to run only this specific test with this specific `STAKE_POOL_PROVIDER_URL` setting.

If in the future we will add some other tests under `test:providers`, they will be executed about 5 lines before in the same file and they will be not required here.

Reply:
3af295122 should solve this